Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import unittest
+
+from Bio.PDB import PDBParser, PSEA
+
+
+os.environ['LANG'] = 'C'
+
+p-sea = None
+from Bio._py3k import getoutput
+try:
+    output = getoutput("p-sea -h")
+    if output.startswith("displays this help"):
+        p-sea = "p-sea"
+except OSError:
+    pass
+
+if not clustalo_exe:
```

Comment:
That should be testing if the p-sea tool was found (see comment about what to use as the variable name).